src/components/layout/navbar/Navbar.content.tsx

```diff
@@ -1,22 +1,22 @@
 export const navbarItems = [
   {
-    tag: "#oplese",
-    text: "O plese",
+    tag: '#oplese',
+    text: 'O plese',
   },
   {
-    tag: "#vstupenky",
-    text: "Vstupenky",
+    tag: '#program',
+    text: 'Program',
   },
   {
-    tag: "#program",
-    text: "Program",
+    tag: '#vstupenky',
+    text: 'Vstupenky',
   },
   {
-    tag: "#kontakt",
-    text: "Kontakt",
+    tag: '#kontakt',
+    text: 'Kontakt',
   },
   {
-    tag: "#partneri",
-    text: "Partneři",
+    tag: '#partneri',
```

Comment:
You could do this with just some .prepend() function and add #

Reply:
I know, but there are special characters, lowercase, and # ... so 2-3 changes, the new prop is better here IMO